.github/workflows/run_integration_tests.yml

```diff
@@ -69,7 +69,9 @@ jobs:
         run: |
           if [ "${{ github.event.inputs.unbounded-test-namespaces }}" == "ALL" ] ; then
             # Use the full list of namespaces
-            namespaces="[ 'CosmosDB', 'Couchbase', 'Elasticsearch', 'MongoDB', 'Msmq', 'MsSql', 'MySql', 'NServiceBus', 'NServiceBus5', 'Oracle', 'Postgres', 'RabbitMq', 'Redis' ]"
+            # Oracle is disabled, see https://issues.newrelic.com/browse/NR-161046
```

Comment:
Are we supposed to include these references?

Reply:
Probably not.  I've removed them.